### PR TITLE
Add dlatk recipe

### PR DESCRIPTION
**Pull request by izahn**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by izahn** [comment]
Holding off on this per discussion at https://github.com/dlatk/dlatk/issues/24

Please do NOT review yet.

**Comment by izahn** [comment]
Ready for review @conda-forge/help-python

File: recipes/dlatk/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  sha256: fc4c8f12c597623facb5100e6d836a191459655938c83ccc352df377bd07cebd
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - numpy
```
Comment (chrisburr) [inline recipes/dlatk/meta.yaml:20]: This probably isn't needed
```suggestion
```

File: recipes/dlatk/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+test:
+  imports:
+    - dlatk
+    - dlatk.lexicainterface
+  requires:
+    - pip
+
+about:
+  home: http://dlatk.wwbp.org
+  summary: DLATK is an end to end human text analysis package, specifically suited for social media and social scientific applications. It is written in Python 3 and developed by the World Well-Being Project at the University of Pennsylvania and Stony Brook University.
```
Comment (chrisburr) [inline recipes/dlatk/meta.yaml:45]: The summary should be short. For a longer bit of text you should use the `description` field as well.

File: recipes/dlatk/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    - scipy >=0.13.3
+    - sqlalchemy >=0.9.9
+    - statsmodels >=0.5.0
+
+test:
+  imports:
+    - dlatk
+    - dlatk.lexicainterface
+  requires:
+    - pip
```
Comment (chrisburr) [inline recipes/dlatk/meta.yaml:41]:
```suggestion
    - pip
  commands:
    - pip check
```

File: recipes/dlatk/meta.yaml (hunk 1 above)
Comment (chrisburr) [reply recipes/dlatk/meta.yaml]: It looks like you actually do need `numpy` as the package pointlessly includes it in `setup_requires`: https://github.com/dlatk/dlatk/blob/87034d79b1f8255ac88e1bbac003e8936e3e2772/setup.py#L149-L151

Comment (izahn) [reply recipes/dlatk/meta.yaml]: yep added it back, thanks!

File: recipes/dlatk/meta.yaml (hunk 2 above)
Comment (izahn) [reply recipes/dlatk/meta.yaml]: fixed in https://github.com/conda-forge/staged-recipes/pull/14068/commits/4a1d6b14270605e94cb55ce8f9633a6b45ed9869

File: recipes/dlatk/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    - pip check
+
+about:
+  home: http://dlatk.wwbp.org
+  summary: End to end human text analysis package.
+  description: |
+    DLATK is an end to end human text analysis package, specifically
+    suited for social media and social scientific applications. It is written in 
+    Python 3 and developed by the World Well-Being Project at the University of 
+    Pennsylvania and Stony Brook University."
```
Comment (chrisburr) [inline recipes/dlatk/meta.yaml:52]:
```suggestion
    Pennsylvania and Stony Brook University.
```

**Pull request merged**